bigtable/spec/filter_samples_spec.rb

```diff
@@ -0,0 +1,613 @@
+Column Family stats_summary
+\tconnected_cell: 0 @#{@timestamp}
+\tconnected_wifi: 1 @#{@timestamp}
+\tos_build: PQ2A.190406.000 @#{@timestamp}
+
+Reading data for phone#5c10102#20190501:
+Column Family cell_plan
+\tdata_plan_10gb: true @#{@timestamp}
+Column Family stats_summary
+\tconnected_cell: 1 @#{@timestamp}
+\tconnected_wifi: 1 @#{@timestamp}
+\tos_build: PQ2A.190401.002 @#{@timestamp}
+
+Reading data for phone#5c10102#20190502:
+Column Family cell_plan
+\tdata_plan_10gb: true @#{@timestamp}
+Column Family stats_summary
+\tconnected_cell: 1 @#{@timestamp}
+\tconnected_wifi: 0 @#{@timestamp}
+\tos_build: PQ2A.190406.000 @#{@timestamp}
+
+"
+  end
+
+  it 'filter_limit_cells_per_row' do
+    output = capture do
+      filter_limit_cells_per_row @project_id, @instance_id, @table_id
+    end
+    expect(output).to match "Reading data for phone#4c410523#20190501:
+Column Family cell_plan
+\tdata_plan_01gb: false @#{@timestamp}
+\tdata_plan_01gb: true @#{@timestamp_minus_hr}
+
+Reading data for phone#4c410523#20190502:
+Column Family cell_plan
+\tdata_plan_05gb: true @#{@timestamp}
+Column Family stats_summary
+\tconnected_cell: 1 @#{@timestamp}
+
+Reading data for phone#4c410523#20190505:
+Column Family cell_plan
+\tdata_plan_05gb: true @#{@timestamp}
+Column Family stats_summary
+\tconnected_cell: 0 @#{@timestamp}
+
+Reading data for phone#5c10102#20190501:
+Column Family cell_plan
+\tdata_plan_10gb: true @#{@timestamp}
+Column Family stats_summary
+\tconnected_cell: 1 @#{@timestamp}
+
+Reading data for phone#5c10102#20190502:
+Column Family cell_plan
+\tdata_plan_10gb: true @#{@timestamp}
+Column Family stats_summary
+\tconnected_cell: 1 @#{@timestamp}
+
+"
+  end
+
+  it 'filter_limit_cells_per_row_offset' do
+    output = capture do
+      filter_limit_cells_per_row_offset @project_id, @instance_id, @table_id
+    end
+    expect(output).to match "Reading data for phone#4c410523#20190501:
+Column Family cell_plan
+\tdata_plan_05gb: true @#{@timestamp}
+Column Family stats_summary
+\tconnected_cell: 1 @#{@timestamp}
+\tconnected_wifi: 1 @#{@timestamp}
+\tos_build: PQ2A.190405.003 @#{@timestamp}
+
+Reading data for phone#4c410523#20190502:
+Column Family stats_summary
+\tconnected_wifi: 1 @#{@timestamp}
+\tos_build: PQ2A.190405.004 @#{@timestamp}
+
+Reading data for phone#4c410523#20190505:
+Column Family stats_summary
+\tconnected_wifi: 1 @#{@timestamp}
+\tos_build: PQ2A.190406.000 @#{@timestamp}
+
+Reading data for phone#5c10102#20190501:
+Column Family stats_summary
+\tconnected_wifi: 1 @#{@timestamp}
+\tos_build: PQ2A.190401.002 @#{@timestamp}
+
+Reading data for phone#5c10102#20190502:
+Column Family stats_summary
+\tconnected_wifi: 0 @#{@timestamp}
+\tos_build: PQ2A.190406.000 @#{@timestamp}
+
+"
+  end
+
+  it 'filter_limit_col_family_regex' do
+    output = capture do
+      filter_limit_col_family_regex @project_id, @instance_id, @table_id
+    end
+    expect(output).to match "Reading data for phone#4c410523#20190501:
```

Comment:
Nit: For multi-line strings, could you do 
`<<~OUTPUT`
`        Reading data for phone`...
`        tos_build: PQ2A.190406.000 @#{@timestamp}`
`OUTPUT`